src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -0,0 +1,156 @@
+package bio.terra.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import bio.terra.cloudfunctions.common.GoogleCloudEventHarness;
+import bio.terra.cloudfunctions.common.GsonWrapper;
+import bio.terra.cloudfunctions.common.MediaTypeWrapper;
+import com.google.cloud.functions.Context;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+import java.io.InputStream;
+import java.util.logging.Logger;
+import org.junit.Before;
+
+public class BaseTest {
+  private static final Logger logger = Logger.getLogger(BaseTest.class.getName());
+
+  protected static String MOCK_GCS_EVENT_GZIP;
+  protected static String MOCK_GCS_EVENT_JSON;
+  protected static GCSCloudEventContext GCS_CLOUD_EVENT_CONTEXT;
+  protected static UnSupportedCloudEventContext UNSUPPORTED_CLOUD_EVENT_CONTEXT;
+  protected static MediaTypeWrapper APPLICATION_GZIP;
+  protected static MediaTypeWrapper APPLICATION_X_GZIP;
+  protected static MediaTypeWrapper APPLICATION_JSON;
+  protected static InputStream MOCK_TGZ;
+  protected static InputStream MOCK_GZ;
+  protected static InputStream MOCK_JSON;
+  protected static byte[] MOCK_STORAGE_OBJECT_DATASTREAM;
+
+  @Before
+  public void setUp() {
+    try {
+      MOCK_GCS_EVENT_GZIP =
+          new String(
+              getClass()
+                  .getClassLoader()
+                  .getResourceAsStream("testfiles/mock_gcs_event_gzip.json")
+                  .readAllBytes());
+
+      MOCK_GCS_EVENT_JSON =
```

Comment:
Does not appear to be used anymore, please delete along with json file.

Reply:
Removed unused test resource files.